spec/syntax.md

```diff
@@ -347,19 +341,27 @@ Whitespace within a _pattern_ is meaningful and MUST be preserved.
 
 ### Expressions
 
-**_Expressions_** MUST start with a _literal_, a _variable_, or an _annotation_.
+**_Expressions_** MUST start with an _operand_ or an _annotation_.
 An _expression_ MUST NOT be empty.
 
-A _literal_ or _variable_ MAY be optionally followed by an _annotation_.
+An **_operand_** is either a _literal_ or a _variable_.
+An _operand_ MAY be optionally followed by an _annotation_.
 
 An **_annotation_** consists of a _function_ and its named _options_,
 or consists of a _reserved_ sequence.
 
 _Functions_ do not accept any positional arguments
-other than the _literal_ or _variable_ in front of them.
+other than the _operand_ in front of them.
+
+_Functions_ use one of the following prefix sigils:
+
+- `:` for standalone content
+- `+` for starting or opening elements
+- `-` for ending or closing elements
```

Comment:
Instead of "elements", say "_expressions_", since that is our name for a placeable element.

Reply:
Done.